doc/source/whatsnew/v0.16.0.txt

```diff
@@ -268,3 +270,10 @@ Bug Fixes
 
 - Bug in ``read_csv`` with buffer overflows with certain malformed input files (:issue:`9205`)
 - Bug in groupby MultiIndex with missing pair (:issue:`9049`, :issue:`9344`)
+
+
+
+
+- Bugs where ``Series.value_counts`` would exclude NaN for categorical ``Series`` even with ``dropna = False``, and include a NaN entry with a count of 0 when it shouldn't (:issue:`9443`)
+- ``Series.value_counts(dropna = False, sort = False)`` and ``Series.describe()`` for categorical ``Series`` are now consistent in putting NaN entires at the end
```

Comment:
- Bug in ``Series.values_counts`` with excluding NaN for categorical type ``Series`` with ``dropna=True``
- ``Series.values_counts`` and ``Series.describe`` for categorical type will now put NaN entries at the end.